test/.phpunit.result.cache

```diff
@@ -0,0 +1 @@
+{"version":1,"defects":{"EnrollmentTest::test_register_enrollment_custom_post_type":4,"App\\tests\\EnrollmentTest::test_register_enrollment_custom_post_type":4,"App\\tests\\enrollment_test::test_register_enrollment_custom_post_type":4,"App\\tests\\Enrollment_Test::test_register_enrollment_custom_post_type":4},"times":{"EnrollmentTest::test_register_enrollment_custom_post_type":0.01,"App\\tests\\EnrollmentTest::test_register_enrollment_custom_post_type":0.363,"App\\tests\\enrollment_test::test_register_enrollment_custom_post_type":0.009,"App\\tests\\Enrollment_Test::test_register_enrollment_custom_post_type":0.033}}
```

Comment:
Do we need to upgrade this file on GitHub?

Reply:
File deleted.